autorest/preparer_test.go

```diff
@@ -70,7 +70,7 @@ func ExampleCreatePreparer() {
 	} else {
 		fmt.Println(r.URL)
 	}
-	// Output: https://microsoft.com/a/b/c
```

Comment:
@garimakhulbe Ok. I'm confused. "https:/a/b/c" is __not__ a valid URL. It lacks the required host component.